src/server/db/seo-queries.ts

```
@@ -1,246 +1,165 @@
-import { getFromCache } from '@/lib/cache/seo-cache'
-import { withMetrics } from '@/lib/monitoring/seo-metrics'
+import { cacheLife, cacheTag } from 'next/cache'
 import { prisma } from '@/server/db'
-import { ms } from '@/utils/time'
 import { ApprovalStatus } from '@orm/client'
```

ESLint

[error] `@orm/client` import should occur before import of `next/cache`

(import/order)